### PR TITLE
Gang Update: Fashion War Edition

### PR DESCRIPTION
**Pull request by Ikarrus**
This update primarily deals with gang outfits and expands on how they affect the round. The basic idea behind this is an extension of the idea that notoriety = influence, and that you should be rewarded for what basically amounts to outing yourself.
##### Major Changes
- When territory is calculated, every gangster wearing their gang's outfit now increases influence by 2 and shortens takeover time by 2 seconds.
- Added several more options for gang outfits
  - Gang Colored Suits
  - Overcoats
  - Turtlenecks
  - Soviet Uniforms
- Gang outfits are now slightly armored (10% melee, bullet, and laser)
- Gang tags are now named "graffiti" to make them harder to detect
##### Minor Changes
- Dominator health increased to 300
- Dominators now set Delta Alert on activation. Delta text has been changed to accommodate this.
- Pinpointers will point towards active dominators
- X-ray laser guns now only deal half damage to dominators
- Initial takeover timer is now capped at 5 minutes (50% control)
- White suits now have blue shirts
- Gang outfits added to clothes vendor
  - Armored overcoat (armor vest values) added to HoS locker

### CODE REVIEW

File: html/changelogs/Ikarrus-gangfashion.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Wearing your gang's outfit now increases influence and shortens takeover time faster."
+  - rscadd: "Added several more options for gang outfits"
+  - tweak: "Gang outfits are now slightly armored"
+  - tweak: "Gang tags are now named "graffiti" to make them harder to detect"
```
Comment (optimumtact) [inline html/changelogs/Ikarrus-gangfashion.yml]: Can't have those extra " in there

either escape them with \ or use ' and ' around this entry

**Comment by Iamgoofball** [comment]
gotta represent your gang now

:+1:

**Comment by Lumipharon** [comment]
Fuck yes.
Need gang patches on the jackets etc.
Could make em an accessory.

**Comment by PKPenguin321** [comment]
Remind me, is this mode in the rotation yet? Because it's fucking radical.

**Comment by Ikarrus** [comment]
odds

**Comment by pudl** [comment]
I don't see why gang members need 5 different variations of outfit.

**Comment by Ikarrus** [comment]
Oh, it's a lot more than 5. 

They provide options and an opportunity to define what your gang is from an RP perspective. Having more variety also makes them less likely to be meta'd out as antag outfits.

File: code/game/gamemodes/gang/dominator.dm [hunk 2]
```diff
@@ -19,7 +19,7 @@
 
 /obj/machinery/dominator/examine(mob/user)
 	..()
-	if(broken)
+	if(operating == -1)
```
Comment (Miauw62) [inline code/game/gamemodes/gang/dominator.dm]: these might be best as #defines

File: code/game/gamemodes/gang/dominator.dm [hunk 3]
```diff
@@ -51,23 +51,22 @@
 
 	health -= damage
 
-	switch(health)
-		if(101 to INFINITY)
-			if(prob(damage*2))
-				sparks.set_up(5, 1, src)
-				sparks.start()
-		if(1 to 100)
+	if(health > (maxhealth/2))
```
Comment (Miauw62) [inline code/game/gamemodes/gang/dominator.dm]: Shouldnt this be < instead of >?

File: code/game/gamemodes/gang/dominator.dm [hunk 4]
```diff
@@ -77,21 +76,33 @@
...
-			priority_announce("All hostile activity within station systems have ceased.","Network Alert")
+	if(gang)
+		if(!isnum(mode.A_timer) && !isnum(mode.B_timer))
+			SSshuttle.emergencyNoEscape = 0
+			if(SSshuttle.emergency.mode == SHUTTLE_STRANDED)
+				SSshuttle.emergency.mode = SHUTTLE_DOCKED
+				SSshuttle.emergency.timer = world.time
+				priority_announce("Hostile enviroment resolved. You have 3 minutes to board the Emergency Shuttle.", null, 'sound/AI/shuttledock.ogg', "Priority")
+			else
+				priority_announce("All hostile activity within station systems have ceased.","Network Alert")
```
Comment (Miauw62) [inline code/game/gamemodes/gang/dominator.dm]: You might want to reset the pinpointer's target in this case.

File: code/game/gamemodes/gang/dominator.dm (hunk 3 above)
Comment (Ikarrus) [inline code/game/gamemodes/gang/dominator.dm]: Nope, it looks fine to me. The icon doesn't start looking damaged until it is less than half health.

File: code/game/gamemodes/gang/dominator.dm (hunk 4 above)
Comment (Ikarrus) [inline code/game/gamemodes/gang/dominator.dm]: Alright, I've updated to make it work this way.

**Comment by pudl** [comment]
@Ikarrus Why don't you just make 1 outfit that can be reskinned like bartender guns and shit.

**Comment by Ikarrus** [comment]
Because gang outfits need to be uniform for my purposes.

**Comment by pudl** [comment]
@Ikarrus Security is one of the most tight-knit groups on station and even they get alt outfits and variation. What makes gangs different?

**Comment by Ikarrus** [comment]
Because stealth is an element of gangs and having outfits be uniform allows me to make them appear to be part of a gang.

**Comment by pudl** [comment]
@Ikarrus In reality, gangs are usually united by a common color as opposed to all of them wearing the same article of clothing. Every other department is united by color as opposed to what kind of outfit they wear. In science, although roboticists have black jumpsuits and scientists have white ones, they both have purple on them.

**Comment by Ikarrus** [comment]
This isn't mean to mimic reality. It serves a gameplay function.

**Comment by pudl** [comment]
@Ikarrus Usually it's encouraged to not stay strict to reality, but this would be a better function than a single uniform. I'd like the idea of a gang wearing multiple types of purple uniforms, instead of multiple colors of turtleneck.

**Comment by Ikarrus** [comment]
Maybe, but I'm opting to have more variety options over using colors right now.

**Comment by pudl** [comment]
@Ikarrus Ok, then you aren't even trying to say why this is better than my idea anymore. Whatever then.

**Comment by Miauw62** [comment]
pudl is the new paprika

**Comment by optimumtact** [comment]
basically

**Pull request merged**